tools/c7n_oci/c7n_oci/output.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+
+import oci
+from oci.exceptions import ServiceError
+
+from c7n.output import blob_outputs, BlobOutput, log_outputs, LogOutput
+from c7n_oci.constants import PROFILE, OCI_LOG_COMPARTMENT_ID
+from c7n_oci.log import OCILogHandler
+from c7n_oci.session import SessionFactory
+
+
+@blob_outputs.register("oci")
+class OCIObjectStorageOutput(BlobOutput):
+    log = logging.getLogger('custodian.oci.output.OCIObjectStorageOutput')
+
+    def __init__(self, ctx, config):
+        super(OCIObjectStorageOutput, self).__init__(ctx, config)
+        if PROFILE in self.config.keys():
+            session = SessionFactory(profile=self.config.profile)()
```

Comment:
SessionFactory(profile=self.config.get('profile')) should allow collapsing this conditional.

Reply:
Updated to use this